doc/vmware_workstation_15_5_0_setup.rst

```diff
@@ -0,0 +1,63 @@
+
+For WMware worktation 15.5.0
+============================
+
+If your installation of VMWare workstation not work with latest version of Windows, you could install old version of WMware workstation.
+
+Here the steps for running `cortx-va-1.0.2.zip </releases/download/VA/cortx-va-1.0.2.zip>`_ for WMware Workstation 15.5.0 in Window 10 version 1890 ( OS build 17763.1637) 
+
+#. Install `WMware Workstation 15.5.0 <https://www.youwindowsworld.com/en/downloads/virtualization/vmware/vmware-workstation-15-pro/download-535-vmware-workstation-15-pro>`_. 
+   (Offial VMWare no longer provided old version for download).
```

Comment:
We should not have this URL (download site) in our document, I suppose.

Reply:
@huanghua78  any suggestion, How about ask the user to find online instead?

Reply:
@huanghua78  I remove the link and ask user download their own instead,

Reply:
The problem is specific for "VMware Workstation 15.5.0"?
If so, a user must have his/her own copy of this software. You don't have to provide a download site.

We have tested and verified with various versions of VMWare.